localnodedns/controllers/tests/simple.out.yaml

```diff
@@ -0,0 +1,235 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+    addons.x-k8s.io/localnodedns: localnodedns-sample
+    k8s-app: localnodedns
+    kubernetes.io/cluster-service: "true"
+  name: node-local-dns
+  namespace: kube-system
+
+---
+
+apiVersion: v1
+data:
+  Corefile: |
+    cluster.local:53 {
+        errors
+        cache {
+                success 9984 30
+                denial 9984 5
+        }
+        reload
+        loop
+        bind 169.254.20.10 10.96.0.10
+        forward . __PILLAR__CLUSTER__DNS__ {
```

Comment:
The `__PILLAR__CLUSTER__DNS__` placeholder values should probably be generated properly

Reply:
> __PILLAR__CLUSTER__DNS__ and __PILLAR__UPSTREAM__SERVERS__ will be populated by the node-local-dns pods

The placeholders that are replaced are __PILLAR__LOCAL__DNS__, __PILLAR__DNS__DOMAIN__, __PILLAR__DNS__SERVER__